docs/ops/doc/IntrospectingOps.md

```diff
@@ -0,0 +1,124 @@
+# Reasoning about Ops
+
+The declarative nature of Ops is convenient for matching and execution, but sometimes it is necessary to know where an Op comes from. This page describes the tools provided by SciJava Ops to understand the code underneath.
+
+Naturally, to understand where an Op came from, you first need an Op:
+
+```java
+var img = ...;
+var out = ...;
+var sigma = 5.0;
+
+// This Op call finds a "filter.gauss Op" that blurs an image using a provided
+// sigma, placing the result in the output container "out".
+//
+// NB the Op is saved within the variable "op"
+var op = ops.op("filter.gauss").input(img, sigma).output(out).computer();
+```
+
+## Introduction
+
+Each Op is generated from an `OpInfo` object, describing an Op's source code, version, and other associated metadata. If you've matched an Op `op`, you can obtain its `OpInfo` using the method `Ops.info`:
+
+```java
+import org.scijava.ops.api.Ops;
+
+var info = Ops.info(op);
+```
+
+The `OpInfo` often provides all needed information, but if `op` utilizes dependencies or other framework features, you may wish to introspect the "rich" Op instead, using the method `Ops.rich`.
+
+```java
+var richOp = Ops.rich(op);
```

Comment:
This is the only mention of rich ops on the page, and there is no explanation of what a "rich" op is nor any examples of API usage.